### ISSUE TITLE
Audited end to end encryption safe against network and server attacks?

### ISSUE THREAD

**Issue by xandro0777** [ArcaneChat/android#35]
Hi, title is a quote from the F-Droid description, however I don't see the same claim in the Readme.

Can you please clarify? Original Delta Chat was not in any way safe against MITM attacks.

**Comment by adbenitez** [comment]
hi @xandro0777 the e2ee in Delta Chat (and any app/client that uses Delta Chat core like ArcaneChat) is safe against MITM attacks thanks to the "protected chats" feature, that is, the green checkmark you see in any chat with a contact you performed a QR/link verification which ensures there is no MITM

you can read more about it offline in the FAQ inside the app or at: https://delta.chat/en/help

Some fragments:

> Chat titles with green checkmarks green checkmark mean that all messages in the chat will be end-to-end encrypted and **can not be read or altered by compromised e-mail servers or Internet providers.**

> [...] Guaranteed end-to-end encrypted chats protect against MITM attacks [...]

**Comment by xandro0777** [comment]
Thanks, I looked up the pointers and see that a lot of work has been done since I last looked at Deltachat/Autocrypt. From deliberately designed to be unsafe to quite the opposite.

The Audit of ETH looks reassuring.

However,  quite a bit of scepticism remains. On the way from something naively insecure to real e2e the complexity grew by some orders of magnitude. There are now quite a few encryption, key exchange etc frameworks in place and such combinations tend to increase attack surface.

Finally.. the claim in the FAQ that it compares favorably to the Signal encryption is very daring given the level of scrutiny Signal did get.

A bit of history.. once upon a time there used to be a perfectly well working email encryption system with PGP strictly relying on out of channel key exchange.. PGP signing parties exchanging hardcopies of keys and keyrings. Then came someone claiming this is too complicated and tried to replace it with a simple but designed to be insecure protocol (Autocrypt). After this was recognized just too inviting for MITM another layer was implemented on top of it actually doing manual key exchange and verification but trying to remain compatible to Autocrypt and everything else. So the design is now back to practically the same as manual PGP key verification 35 years ago but with several additional layers of protocols and many more software components involved and attempting to be compatible with other apps.
